src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java

```diff
@@ -3632,10 +3621,14 @@ public DialogueNodeType getDialogueNodeType() {
 	
 	
 	private static String getFetishEntry(Fetish othersFetish, Fetish selfFetish) {
-		return "<div class='container-full-width' style='background:transparent; margin:2px 0; width:100%;'>"
-					+ getIndividualFetishEntry(othersFetish)
-					+ (selfFetish==null?"":getIndividualFetishEntry(selfFetish))
-				+ "</div>";
+		if(othersFetish.isContentEnabled()) {
```

Comment:
So far, the others and self entries have always occurred in pairs, when it comes to fetish settings.

But what if in the future, we add content settings for the bimbo and crossdresser fetish, for example? (Or add new unpaired fetishes with settings.)

Maybe add a gate to `getIndividualFetishEntry` for when single fetishes are disabled, and also use a condition here like:
```java
if(othersFetish.isContentEnabled() && (selfFetish == null || selfFetish.isContentEnabled())) {
```

Reply:
There are actually 3 fetishes that don't have pairings already, see `PhoneDialogue.java:3562-3573` of which 2 can be content blocked
Rather than make something complicated to address this for now I've left the existing code in place

Reply:
I see, yeah, sounds good.

I have a few ideas for how to streamline this area a bit, but it can wait.

Reply:
I have slept, and I have an idea on how to resolve this
Going to add pairings to the fetish object as they are used in a few locations (Phone dialog, Attacks, assigning fetishes at random) and then change the fetish menu at `PhoneDialogue.java:3541` into a loop so pairs and individual fetishes alike are handled cleanly without having to add them manually